Kickstarter-iOS/ViewModels/AppDelegateViewModel.swift

```diff
@@ -403,27 +403,71 @@ public final class AppDelegateViewModel: AppDelegateViewModelType, AppDelegateVi
     self.findRedirectUrl = deepLinkUrl
       .filter { Navigation.match($0) == .emailClick }
 
+    /// INFO: Handles the deeplink route with both an id and text based name for a deeplink for categories.
```

Comment:
Haven't seen us use this INFO prefix before so I think the /// with your inline doc is fine.